### ISSUE TITLE
Violation about Preventing invalid impressions

### ISSUE THREAD

**Issue by AmiriOS2019** [HMS-Core/hms-ads-demo-java#20]
I used the same code demo from Native Ads in my app and the banner displays correctly but when I published it on AppGallery I got the following message: 

An ad cannot be displayed repeatedly for one request. After being displayed, the ad needs to be loaded again in real time for the next request.
[Recommended] The ad object on each page is independent. You need to use the createXX method to create ads and register a callback.

Please, has anyone had the same problem with this code being used?

**Comment by ttljtw** [comment]
Yes, it is recommended to create a new one at a time, as in the [demo](https://github.com/HMS-Core/hms-ads-demo-java/blob/6ed3c45cf96344555d5419624760b3098b3506d5/app/src/main/java/com/huawei/hms/ads/sdk/BannerActivity.java#L64).

**Comment by AmiriOS2019** [comment]
I updated my code today and the ads are showing correctly on the phone. but i'm not sur if it's the correct way to avoid the message : An ad cannot be displayed repeatedly for one request

Please have a look [https://github.com/AmiriOS2019/myfiles2/blob/main/MyActivity.java](url)

**Comment by ttljtw** [comment]
Look at the code and I think your problem can be solved. 
But, I don't think it's handled right [here](https://github.com/AmiriOS2019/myfiles2/blob/076ed12ba11671ffed5c50c13624cf9e40064a05/MyActivity.java#L61), why did you destory ad just loaded?

**Comment by AmiriOS2019** [comment]
Thank you ttljtw for your remark. I put it first like on the demo code [link](https://github.com/HMS-Core/hms-ads-demo-java/blob/master/app/src/main/java/com/huawei/hms/ads/sdk/NativeActivity.java#L152), but after seeing the documentation I noticed that I should destroy the native ad after displaying it, so I move it after adScrollView.addView(nativeView);
